sample_data/contiguous_usa_by_county_since_2001/contiguous_usa_by_county_since_2001.md

```diff
@@ -0,0 +1,31 @@
+# Sources
+
+## contiguous_usa_by_county_since_2001.geojson
+United States Census Bureau. Cartographic Boundary Files.
+Downloaded from: https://www2.census.gov/geo/tiger/GENZ2021/shp/cb_2021_us_county_500k.zip on 18 July 2022.
+Lines within the .json file containing shapes for Alaska, Puerto Rico, US Virgin Islands, American Samoa, Northern Mariana Islands, Guam, and US Virgin Islands were manually deleted.
+
+### Code for simplification
+```
+usa <- geojson_sf("~/contiguous_usa_by_county_since_2001/cb_2021_us_county_500k.json")
+target_n_pts_in_output <- 48500
```

Comment:
This is for further discussion during the meeting.

Since the map is huge, let us think about whether we should simplify it to only 48500 points or even more.